doc/developers/contributing.rst

```diff
@@ -779,6 +788,56 @@ cross-compatibility and is included in scikit-learn as
 ``sklearn.externals.six``.
 
 
+.. _code_review:
+
+Code Review Guidelines
+======================
+Reviewing code contributed to the project as PRs is a crucial component
+of scikit-learn development. We encourage anyone to start reviewing code of other
+developers. The code review process is often highly educational for everybody involved.
+While each pull request needs to be signed off by two core developers, you can
+speed up this process by providing your feedback.
+
+Here are a few important aspects that need to be covered in any code review,
+from high-level questions to a more detailed check-list.
+
+- Do we want this in the library? Is it likely to be used? Is it in the scope
+  of scikit-learn and does the benefit of the addition warrant the maintenance burden?
```

Comment:
" and" -> "?"

I think we should avoid "warrant" and "burden" as difficult words. "Will the cost of maintaining a new feature be worth its benefits?" might be a rephrasing in simpler english.